app/uk/gov/hmrc/apidocumentation/controllers/ApiDocumentationController.scala

```diff
@@ -261,7 +263,7 @@ class ApiDocumentationController @Inject()(
         apidocumentation.models.PageAttributes(apiDefinition.name, breadcrumbs, navLinks, navigationService.sidebarNavigation())
       }
 
-      XmlApiDocumentation.xmlApiDefinitions.find(_.name == name) match {
+      xmlServicesService.fetchXmlApi(name) flatMap {
```

Comment:
Map with no need for Future.successful in both case statements?

Reply:
Yep. I'll change it